specification/workloadmonitor/resource-manager/microsoft.workloadmonitor/preview/2018-08-31-preview/Microsoft.WorkloadMonitor.json

```diff
@@ -601,9 +601,6 @@
           },
           {
             "$ref": "#/parameters/api-version"
-          },
-          {
```

Comment:
@vivek-microsoft Why are the skipToken and nxtlink params being pulled out? Does the API not support pagination?